### ISSUE TITLE
Show in preview if delayRender() is being called

### ISSUE THREAD

**Issue by JonnyBurger** [remotion-dev/remotion#674]
Would be nice to visually indicate that a delayRender() handle is being open.

**Comment by musafiroon** [comment]
Should also show something in remotion studio when delayRender is called

**Comment by JonnyBurger** [comment]
@musafiroon Agreed, it's the same thing, Studio is just the new name for Preview.

**Comment by JonnyBurger** [comment]
Not planning after all

- Unless we hide it somewhere, it probably is visual noise
- Agents usually don't make this mistake
- We have good debugability for renders failing with delayRender now
